security/bitwarden-cli/Portfile

```diff
@@ -16,10 +16,6 @@ description         Bitwarden password manager CLI
 long_description    CLI implementation of the Bitwarden password manager.
 homepage            https://bitwarden.com
 
-checksums           rmd160  5eb1201c952bc4e4e6da8df7be4528773cbb97db \
-                    sha256  6de961d51e5605b29e73be616c7b071fe6464d4cf87af870ff18b043953b2eab \
-                    size    200634
-
```

Comment:
why are the checksums now gone?

Reply:
Because checksums are not required / checked when using `fetch.type git`. I overlooked this in https://github.com/macports/macports-ports/pull/12675.